tools/wt_verify/wt_verify.py

```diff
@@ -586,12 +593,17 @@ def main():
 
     parsed_data = None
 
-    if "dump_pages" in command:
-        parsed_data = parse_dump_pages()
-    elif "dump_blocks" in command:
-        parsed_data = parse_dump_blocks()
-    else:
-        assert False, f"Unexpected command '{command}'"
+    try:
+        if "dump_pages" in command:
+            parsed_data = parse_dump_pages()
+        elif "dump_blocks" in command:
+            parsed_data = parse_dump_blocks()
+        else:
+            raise Exception(f"Unexpected command '{command}'")
+    except Exception as e:
+        print(str(e), file=sys.stderr)
+        print(traceback.format_exc(), file=sys.stderr)
+        sys.exit(1)
```

Comment:
I don't quite get the logic here - why not remove the `try/except` wrapping and let the exception bubble back up? That way you should get the backtrace for free, and it'll still print the error (I think).

Reply:
Yes the `try/except` is not necessary, I have removed it, thank you!